UI/window-basic-filters.hpp

```diff
@@ -71,6 +71,9 @@ class OBSBasicFilters : public QDialog {
 	void FilterNameEdited(QWidget *editor, QListWidget *list);
 
 	bool isAsync;
+	bool drawablePreview;
```

Comment:
Seems unnecessary to store this here (only "external" user is https://github.com/obsproject/obs-studio/pull/1332/files#diff-49c79ab337c1b285fdee4bfa2b29ee6bR128), because you can easily "recompute" it around https://github.com/obsproject/obs-studio/pull/1332/files#diff-49c79ab337c1b285fdee4bfa2b29ee6bR190 (i.e. make it local like https://github.com/obsproject/obs-studio/pull/1332/files#diff-8b48bed5a6b15dfc1cdbc79269834540R122)